app/src/main/java/com/onelist/RetrofitClientInstance.kt

```diff
@@ -5,14 +5,12 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "" //grocery list website base url
+    private val BASE_URL = ""
 
     val retrofitInstance : Retrofit?
         get() {
-            // has this object been created yet?
             if (retrofit == null) {
-                // create it
-                retrofit = retrofit2.Retrofit.Builder()
```

Comment:
You were not using retrofit2, so I removed it.

Reply:
While too much documentation isn't good, the comments aren't necessarily hurting anything... especially around items that are incomplete.

I do encourage the group to keep work in progress in a separate branch, and only merge to main when complete.